Bottom: f1f46a9fecbb2ddf69ede5f91adac965d372d981
Top:    77707cc185a5324a4cdfe0cae9742364abc337aa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 00:08:37 +0100

find.c: Fixup warnings due to assigned but unused variables


---

diff --git a/src/find.c b/src/find.c
index c5159ba..92c8b35 100644
--- a/src/find.c
+++ b/src/find.c
@@ -1927,13 +1927,8 @@ static bool
 LookupLOConnectionsToArc (ArcTypePtr Arc, Cardinal LayerGroup)
 {
   Cardinal entry;
-  LocationType xlow, xhigh;
   struct lo_info info;
 
-  /* the maximum possible distance */
-  xlow = Arc->BoundingBox.X1 - MAX (MAX_PADSIZE, MAX_LINESIZE) / 2;
-  xhigh = Arc->BoundingBox.X2 + MAX (MAX_PADSIZE, MAX_LINESIZE) / 2;
-
   info.arc = *Arc;
   EXPAND_BOUNDS (&info.arc);
   /* loop over all layers of the group */
